pycds/climate_baseline_helpers.py

```diff
@@ -0,0 +1,169 @@
+            publish=True,
+            # color = '#??????', # TODO: Does this need to be defined?
+        )
+        session.add(network)
+        session.flush()
+    return network
+
+
+def create_pcic_climate_baseline_variables(session):
+    """Create, if they do not exist, the derived variables for climate baseline values.
+    Create the necessary synthetic network for them if it does not already exist.
+
+    Args:
+        session (...): SQLAlchemy session for accessing the database
+
+    Returns:
+        None
+    """
+
+    network = get_or_create_pcic_climate_variables_network(session)
+
+    temp_unit = 'celsius'
+    temp_standard_name = 'air_temperature'
+    precip_unit = 'mm'
+    precip_standard_name = 'lwe_thickness_of_precipitation_amount'
+
+    variable_specs = [
+        {
+            'name': 'Tx_Climatology',
+            'unit': temp_unit,
+            'standard_name': temp_standard_name,
+            'cell_method': 't: maximum within days t: mean within months t: mean over years',
+            'description': 'Climatological mean of monthly mean of maximum daily temperature',
+            'display_name': 'Temperature Climatology (Max.)'
+        },
+        {
+            'name': 'Tn_Climatology',
+            'unit': temp_unit,
+            'standard_name': temp_standard_name,
+            'cell_method': 't: minimum within days t: mean within months t: mean over years',
+            'description': 'Climatological mean of monthly mean of minimum daily temperature',
+            'display_name': 'Temperature Climatology (Min.)'
+        },
+        {
+            'name': 'Precip_Climatology',
+            'unit': precip_unit,
+            'standard_name': precip_standard_name,
+            'cell_method': 't: sum within months t: mean over years',
+            'description': 'Climatological mean of monthly total precipitation',
+            'display_name': 'Precipitation Climatology'
+        },
+    ]
+
+    for vs in variable_specs:
+        variable = session.query(Variable).filter(Variable.name == vs['name']).first()
+        if not variable:
+            vs.update(short_name='{0} {1}'.format(vs['standard_name'], vs['cell_method']),
+                      network_id=network.id)
+            session.add(Variable(**vs))
+    session.flush()
+
+
+# Names and widths of fields in flat source files. Copied from Faron's R code.
+# native_id is the native_id recorded in the database for the station
+# no history record is given but can use latest for the identified station
+field_names = ['native_id', 'assay_flag', 'station_name', 'elev', 'elevation_flag', 'long', 'lat',
+               '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', 'annual']
+field_widths = [8, 1, 12, 5, 1, 12, 12, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6]
+
+# Format string for module `struct`
+field_format = ' '.join(['{}s'.format(fw) for fw in field_widths])
+
+
+def load_pcic_climate_baseline_values(session, var_name, source):
```

Comment:
If  ``source`` is typically the result of ``file.readlines()`` then how about naming the variable ``lines``?